pommerman/graphics.py

```diff
@@ -0,0 +1,327 @@
+    6: { 'id': 6, 'file_name': 'ExtraBomb.png', 'name': 'ExtraBomb', 'image': None},
+    7: { 'id': 7, 'file_name': 'IncrRange.png', 'name': 'IncrRange', 'image': None},
+    8: { 'id': 8, 'file_name': 'Kick.png', 'name': 'Kick', 'image': None},
+    9: { 'id': 9, 'file_name': 'Skull.png', 'name': 'Skull', 'image': None},
+    10: { 'id': 10, 'file_name': 'AgentDummy.png', 'name': 'AgentDummy', 'image': None},
+    11: { 'id': 11, 'file_name': 'Agent0.png', 'name': 'Agent0', 'image': None},
+    12: { 'id': 12, 'file_name': 'Agent1.png', 'name': 'Agent1', 'image': None},
+    13: { 'id': 13, 'file_name': 'Agent2.png', 'name': 'Agent2', 'image': None},
+    14: { 'id': 14, 'file_name': 'Agent3.png', 'name': 'Agent3', 'image': None},
+    15: { 'id': 15, 'file_name': 'AgentDummy-No-Background.png', 'name': 'AgentDummy-No-Background', 'image': None},
+    16: { 'id': 16, 'file_name': 'Agent0-No-Background.png', 'name': 'Agent0-No-Background', 'image': None},
+    17: { 'id': 17, 'file_name': 'Agent1-No-Background.png', 'name': 'Agent1-No-Background', 'image': None},
+    18: { 'id': 18, 'file_name': 'Agent2-No-Background.png', 'name': 'Agent2-No-Background', 'image': None},
+    19: { 'id': 19, 'file_name': 'Agent3-No-Background.png', 'name': 'Agent3-No-Background', 'image': None},
+    20: { 'id': 20, 'file_name': 'X-No-Background.png', 'name': 'X-No-Background', 'image': None}
+}
+
+FONTS_FILE_NAMES = [
+    'Cousine-Regular.ttf'
+]
+
+BORDER_SIZE = 20
+MARGIN_SIZE = 10
+TILE_SIZE = 50
+RESOURCE_DIR = 'resources/'
+BACKGROUND_COLOR = (41, 39, 51, 255)
+TILE_COLOR = (248, 221, 82, 255)
+TEXT_COLOR = (170, 170, 170, 255)
+
+HUMAN_FACTOR = 32
+AGENT_COLORS = [[231,76,60], [46,139,87], [65,105,225], [238,130,238]] # color for each of the 4 agents
+ITEM_COLORS = [[240,248,255], [128,128,128], [210,180,140], [255, 153, 51], [241, 196, 15], [141, 137, 124]]
+ITEM_COLORS += [(153, 153, 255), (153, 204, 204), (97, 169, 169), (48, 117, 117)] # ExtraBomb, IncrRange, etc. 
+
+__location__ = os.path.dirname(os.path.realpath(__file__))
+_resource_path = os.path.join(__location__, RESOURCE_DIR)
+
+class PixelViewer(object):
+    def __init__(self, display=None, board_size=13, agents=[], partially_observable=False):
+        display = rendering.get_display(display)
+
+        self._board_size = board_size
+        self._board_state = None
+        self._agent_count = len(agents)
+        self._agents = agents
+        self._is_partially_observable = partially_observable
+        self._viewer = None
+
+
+    def set_board(self, state):
+        self._board_state = state
+
+
+    def set_agents(self, agents):
+        self._agents = agents
+
+
+    def render(self):
+        viewer = self._viewer
+
+
+    def save(self, path):
+        from PIL import Image
+        img = []
+        Image.fromarray(img).save(path)
+
+
+class PommeViewer(object):
+    def __init__(self, 
+                display=None, 
```

Comment:
Is the display always None?

Reply:
It is in ever example online I have seen, but I have never seen it used when initing the class... I do not have a preference. It was a pattern I copied from the internet.